l10n_it_ddt/models/stock_picking_package_preparation.py

```diff
@@ -287,19 +294,6 @@ def _prepare_invoice(self):
         """
         self.ensure_one()
         order = self._get_sale_order_ref()
-        if order:
```

Comment:
This would break integration with SO.
See https://github.com/OCA/l10n-italy/pull/542